data_mappings.bal

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/lang.'xml as xmllib;
+import ballerina/xmlutils;
+
+# Convert batch event to json
+#
+# + batchEvent - batch event 
+# + return - Return eventhub formatted json
+isolated function getBatchEventJson(BatchEvent batchEvent) returns json {
+    json[] message = [];
+    foreach var item in batchEvent.events {
+        json data = checkpanic item.data.cloneWithType(json);
+        json j = {
```

Comment:
Json j can be used with better variable name?

Reply:
Fixed. Thank you.